nh_clinical/patient.py

```diff
@@ -84,6 +84,16 @@ def _get_name(self, cr, uid, ids, fn, args, context=None):
             result[r['id']] = self._get_fullname(r)
         return result
 
+    def name_get(self, cr, uid, ids, context=None):
```

Comment:
Why `name_get` instead of `get_name`? Is that your preferred naming convention? E.g. *noun_verb* rather than *verb_noun*.